dev-packages/e2e-tests/test-applications/node-otel-custom-sampler/playwright.config.mjs

```diff
@@ -0,0 +1,28 @@
+import { getPlaywrightConfig } from '@sentry-internal/test-utils';
+
+const config = getPlaywrightConfig(
+  {
+    startCommand: `pnpm start`,
+  },
+  {
+    webServer: [
+      {
+        command: `node ./start-event-proxy.mjs`,
+        port: 3031,
+        stdout: 'pipe',
+        stderr: 'pipe',
+      },
+      {
+        command: 'pnpm start',
+        port: 3030,
+        stdout: 'pipe',
+        stderr: 'pipe',
+        env: {
+          PORT: 3030,
+        },
+      },
+    ],
+  },
+);
```

Comment:
no need for this, we only need this if we want to start more than the two default servers - you should be able to just remove the second argument passed to `getPlaywrightConfig`! :)